Frends.PDF.Create/Frends.PDF.Create.Tests/UnitTests.cs

```diff
@@ -0,0 +1,139 @@
+            _footer = new PageContentElement { ContentType = ElementType.Footer, FontFamily = "Times New Roman", FontSize = 8, FontStyle = FontStyleEnum.Regular, LineSpacingInPt = 11, ParagraphAlignment = ParagraphAlignmentEnum.Center, SpacingAfterInPt = 0, SpacingBeforeInPt = 8, HeaderFooterStyle = HeaderFooterStyleEnum.TextPagenum, BorderWidthInPt = 0.0 };
+            _title = new PageContentElement { ContentType = ElementType.Paragraph, FontFamily = "Times New Roman", FontSize = 16, FontStyle = FontStyleEnum.Bold, LineSpacingInPt = 11, ParagraphAlignment = ParagraphAlignmentEnum.Left, SpacingAfterInPt = 0, SpacingBeforeInPt = 8 };
+            _paragraphContent = new PageContentElement { ContentType = ElementType.Paragraph, FontFamily = "Times New Roman", FontSize = 11, FontStyle = FontStyleEnum.Regular, LineSpacingInPt = 11, ParagraphAlignment = ParagraphAlignmentEnum.Left, SpacingAfterInPt = 0, SpacingBeforeInPt = 8 };
+
+            var tablePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"../../../Files/ContentDefinition.json");
+            var tableDefinition = File.ReadAllText(tablePath);
+            _tableContent = new PageContentElement { ContentType = ElementType.Table, Table = tableDefinition };
+
+            _options = new Options { ThrowErrorOnFailure = true };
+        }
+
+        [TearDown]
+        public void TestTearDown()
+        {
+            Directory.Delete(_folder, true);
+        }
+
+        private Result CallCreatePdf(PageContentElement[] contents)
+        {
+            return PDF.Create(_fileProperties, _docSettings, new DocumentContent { Contents = contents }, _options);
+        }
+
+        [Test]
+        public void WritePDF()
+        {
+            _fileProperties.FileExistsAction = FileExistsActionEnum.Overwrite;
+
+            _header.Text = @"This is a header";
+            _footer.Text = @"This is a footer";
+            _title.Text = @"The Document Title";
+            _paragraphContent.Text = @"Some text           for testing
+with some tab
+    one
+        two
+            three. Then end with scandic letter ö and russian word код.";
+
+            var result = CallCreatePdf(new PageContentElement[] { _header, _footer, _title, _paragraphContent, new PageContentElement { ContentType = ElementType.PageBreak }, _tableContent });
+
+            Assert.IsTrue(File.Exists(_destinationFullPath));
+            Assert.IsTrue(result.Success);
+        }
+
+        [Test]
+        public void WritePdf_DoesNotFailIfContentIsEmpty()
+        {
+            _paragraphContent.Text = null;
+            var result = CallCreatePdf(new PageContentElement[] { _paragraphContent });
+
+            Assert.IsTrue(File.Exists(_destinationFullPath));
+            Assert.IsTrue(result.Success);
+        }
+
+        [Test]
+        public void WritePdf_ThrowsExceptionIfFileExists()

```

Comment:
Shouln't all tests be for Create, not for WritePdf (maybe comes from prev codebase?).